pom.xml

```diff
@@ -67,7 +68,7 @@
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
-            <version>[2.15.0,)</version>
+            <version>2.17.1</version>
```

Comment:
Could you make the same change in https://github.com/rovio/rovio-ingest/blob/78b412d4c6871021d9738182fbd16d728a81441e/examples/rovio-ingest-maven-example/pom.xml#L49